components/uuf-core/src/main/java/org/wso2/carbon/uuf/api/config/Configuration.java

```diff
@@ -45,8 +47,8 @@
     private Map<Integer, String> errorPageUris;
     private String defaultErrorPageUri;
     private ListMultimap<String, MenuItem> menus;
-    private Set<String> csrfIgnoreUris;
-    private Set<String> xssIgnoreUris;
+    private Set<UriPatten> csrfIgnoreUris;
```

Comment:
The reason why Lists are suggested here because we might need to maintain an order of the UriPatterns. As per the offline discussion with @this we decided to proceed with Sets. If we come across any requirement to maintain the order of the UriPatterns we will change the Set to List